index.html

```diff
@@ -14,18 +14,62 @@
   <link rel="stylesheet" href="https://cdn.rawgit.com/konpa/devicon/df6431e323547add1b4cf45992913f15286456d3/devicon.min.css">
   <link href="https://fonts.googleapis.com/css?family=Gloria+Hallelujah|Varela+Round" rel="stylesheet">
   <link rel="stylesheet" type="text/css" href="css/style.css" />
-  <title>Nonprofit Product Landing Page Design - Lunatic Apps</title>
+  <!-- Font -->
+  <link href="https://fonts.googleapis.com/css?family=Muli:400,700" rel="stylesheet">
+  <!-- Animate CSS -->
+  <link rel="stylesheet" href="css/animate.css">
 
 </head>
   <body>
-
+    <!-- START SECTION FEATURE#13, BUILT BY MARCIN WICHA-->
+    <div class="row section-features">
+      <div class="col-12">
+        <h2 class="feature-title">Why Choose Lunatic</h2>
+      </div>
+    </div>
+    <div class="row feature-cards-container justify-content-around">
+      <div class="col-md-4 col-12 wow fadeInUp" data-wow-delay="0.3s" style="visibility: visible; animation-delay: 0.3s; animation-name: fadeInUp;">
```

Comment:
Please try not to use inline css. Use your styles into external css style sheet. @marcin-wicha